x/emissions/module/rewards/topic_rewards.go

```diff
@@ -133,7 +85,8 @@ func PickChurnableActiveTopics(
 		// Loop over and run epochs on topics whose inferences are demanded enough to be served
 		// Check the cadence of inferences, and just in case also check multiples of epoch lengths
 		// to avoid potential situations where the block is missed
-		if k.CheckWorkerOpenCadence(block, topic) {
+		//if k.CheckWorkerOpenCadence(block, topic)
```

Comment:
Think this is right. Just should specify why in the function declaration -- because the `block` arg is used to get active topics as of that block